site2/docs/deploy-bare-metal.md

```diff
@@ -76,14 +76,14 @@ For machines running a bookie and a Pulsar broker, more powerful machines are re
 
 ## Install the Pulsar binary package
 
-> You need to install the Pulsar binary package on *each machine in the cluster*, including machines running [ZooKeeper](#deploy-a-zookeeper-cluster) and [BookKeeper](#deploy-a-bookkeeper-cluster).
+> You need to install the Pulsar binary package on each machine in the cluster, including machines running [ZooKeeper](#deploy-a-zookeeper-cluster) and [BookKeeper](#deploy-a-bookkeeper-cluster).
```

Comment:
```suggestion
> You need to install the Pulsar binary package on each machine in the cluster, including machines running ZooKeeper and BookKeeper.
```

Reply:
we don't need to add link here.
Here "ZK" and "BK" are only component, we add link to how to deploy them. It's easily found in the navigation bar.